client/dashboard/app/queries/me-active-subscriptions.ts

```diff
@@ -0,0 +1,8 @@
+import { queryOptions } from '@tanstack/react-query';
+import { fetchActiveSubscriptionsForUser } from '../../data/me-active-subscriptions';
+
+export const activeSubscriptionsQuery = ( { siteId }: { siteId?: string | number } ) =>
```

Comment:
Team review: siteId should have the type `number`. By ensuring it's a number, we reduce missed caching. For instance, queryKey [ `site`, 123 ] will not match with [ `site`, `a.example.com` ].

In this case, using siteSlug might make more sense so that users can see their site slug in the URL instead of the site ID.

Reply:
In that case could we rename it to siteIdOrSlug?

Reply:
I agree to keep it as `siteId: number` for now to avoid leaking complexity to existing code... and we can follow-up later if we want to show site slug in the URL. 🙂